tests/system/provisioning/vars/aws.yml

```diff
@@ -52,20 +52,19 @@ kibana_instance_type: t2.large
 ### Stack variables
 
 # Configuration parameters
-stack_name: fim-tests-1
```

Comment:
This variable is being used at https://github.com/wazuh/wazuh-qa/blob/07624c6055c6aa7873de92209791b3fa41d86f36/tests/system/provisioning/roles/aws/tasks/launch_template.yml#L31

Reply:
Referred to https://github.com/wazuh/wazuh-qa/pull/509#discussion_r382036642